Docker installation seems to be broken

Running `docker build -t mycroft/mimic2:gpu -f gpu.Dockerfile .`
```
...
...
Installing collected packages: audioread, numpy, scipy, joblib, llvmlite, numba, resampy, librosa, matplotlib, tqdm, seaborn, click, itsdangerous, Werkzeug, Jinja2, flask, flask-cors
  Running setup.py install for audioread: started
    Running setup.py install for audioread: finished with status 'done'
  Found existing installation: numpy 1.14.2
    Uninstalling numpy-1.14.2:
      Successfully uninstalled numpy-1.14.2
  Found existing installation: scipy 1.0.1
    Uninstalling scipy-1.0.1:
      Successfully uninstalled scipy-1.0.1
  Running setup.py install for llvmlite: started
    Running setup.py install for llvmlite: finished with status 'error'
    Complete output from command /usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-xvgrgmko/llvmlite/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-8lmz49r4/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    got version from file /tmp/pip-install-xvgrgmko/llvmlite/llvmlite/_version.py {'full': '26059d238f4ba23dff74703dd27168591d889edd', 'version': '0.32.0'}
    running build_ext
    /usr/bin/python3 /tmp/pip-install-xvgrgmko/llvmlite/ffi/build.py
    LLVM version... Traceback (most recent call last):
      File "/tmp/pip-install-xvgrgmko/llvmlite/ffi/build.py", line 106, in main_posix
        out = subprocess.check_output([llvm_config, '--version'])
      File "/usr/lib/python3.5/subprocess.py", line 626, in check_output
        **kwargs).stdout
      File "/usr/lib/python3.5/subprocess.py", line 693, in run
        with Popen(*popenargs, **kwargs) as process:
      File "/usr/lib/python3.5/subprocess.py", line 947, in __init__
        restore_signals, start_new_session)
      File "/usr/lib/python3.5/subprocess.py", line 1551, in _execute_child
        raise child_exception_type(errno_num, err_msg)
    FileNotFoundError: [Errno 2] No such file or directory: 'llvm-config'
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/tmp/pip-install-xvgrgmko/llvmlite/ffi/build.py", line 168, in <module>
        main()
      File "/tmp/pip-install-xvgrgmko/llvmlite/ffi/build.py", line 158, in main
        main_posix('linux', '.so')
      File "/tmp/pip-install-xvgrgmko/llvmlite/ffi/build.py", line 109, in main_posix
        "to the path for llvm-config" % (llvm_config,))
    RuntimeError: llvm-config failed executing, please point LLVM_CONFIG to the path for llvm-config
    error: command '/usr/bin/python3' failed with exit status 1
    
    ----------------------------------------
Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-xvgrgmko/llvmlite/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-8lmz49r4/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-install-xvgrgmko/llvmlite/
You are using pip version 10.0.1, however version 20.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
The command '/bin/sh -c pip install  --no-cache-dir -r requirements.txt' returned a non-zero code: 1

```

Checking on https://apt.llvm.org/ - there doesn't seem to be an apt source for versions 7.x and 8.x

Checking  https://github.com/numba/llvmlite - and although the Compatibility section states: 

> As of version 0.29.0, llvmlite requires LLVM 7.0.x or later

The actual requirement is:

> RuntimeError: Building llvmlite requires LLVM 7.0.x, 7.1.x or 8.0.x

More information can be found in this ticket: https://github.com/numba/llvmlite/issues/578

I see the same bug.

> I fixed this with

Unfortunately, that only gave me another error:
```
  Running setup.py install for llvmlite: started
    Running setup.py install for llvmlite: finished with status 'error'
    ...
    got version from file /tmp/pip-install-fxzdhco_/llvmlite/llvmlite/_version.py {'full': 'aa11b129c0b55973067422397821ae6d44fa5e70', 'version': '0.32.1'}
    running build_ext
    /usr/bin/python3 /tmp/pip-install-fxzdhco_/llvmlite/ffi/build.py
    LLVM version... # static-libstdc++ avoids runtime dependencies on a
    # particular libstdc++ version.
    g++  -shared   -I/usr/lib/llvm-8/include -std=c++11 -fno-exceptions -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -fno-rtti -g -flto assembly.cpp bitcode.cpp core.cpp initfini.cpp module.cpp value.cpp executionengine.cpp transforms.cpp passmanagers.cpp targets.cpp dylib.cpp linker.cpp object_file.cpp -o libllvmlite.so  -L/usr/lib/llvm-8/lib -flto -Wl,--exclude-libs=ALL -lLLVM-8
    /usr/bin/ld: /tmp/cc8TzrVh.ltrans0.ltrans.o: relocation R_X86_64_32S against `_ZN4llvm3sys14DynamicLibrary7InvalidE' can not be used when making a shared object; recompile with -fPIC
    /tmp/cc8TzrVh.ltrans0.ltrans.o: error adding symbols: Bad value
    collect2: error: ld returned 1 exit status
    Makefile.linux:20: recipe for target 'libllvmlite.so' failed
    make: *** [libllvmlite.so] Error 1
    8.0.0
    
    SVML not detected
...
The command '/bin/sh -c LLVM_CONFIG=/usr/lib/llvm-8/bin/llvm-config pip install --no-cache-dir -r requirements.txt' returned a non-zero code: 1
```

--- a/cpu.Dockerfile
+++ b/cpu.Dockerfile
@@ -3,6 +3,8 @@ FROM tensorflow/tensorflow:1.8.0-py3
 RUN mkdir /root/mimic2
 COPY . /root/mimic2
 WORKDIR /root/mimic2
-RUN pip install  --no-cache-dir -r requirements.txt
+RUN apt-get update -y && apt-get install -y llvm-8
+RUN ln -s /usr/bin/llvm-config-8 /usr/bin/llvm-config
+RUN pip install --no-cache-dir -r requirements.txt
 
 ENTRYPOINT [ "/bin/bash" ]

--- a/requirements.txt
+++ b/requirements.txt
@@ -8,3 +8,6 @@ tqdm==4.11.2
 seaborn
 flask_cors
 flask
+# Workaround llvmlite being incompatible with current llvm - Bug #49
+numba==0.45.1
+llvmlite==0.30.0

